palo_alto_pan_os/plugin.spec.yaml

```
@@ -831,3 +831,48 @@ actions:
         description: Was operation successful
         type: boolean
         required: true
+  add_address_object_to_group:
+    title: Add Address Object to Group
+    description: Adds an address object to an address group
+    input:
+      address_object:
+        title: Address Object
+        description: The name of the address object to add
+        type: string
+        example: Malicious IP
```

```suggestion
        example: Malicious IP address
```

---

I think this one is better to avoid confusion

```suggestion
        example: MaliciousHost
```